psi4/src/psi4/scfgrad/response.cc

```diff
@@ -125,16 +125,16 @@ std::shared_ptr<Matrix> RSCFDeriv::hessian_response()
         psio_address next_Smi = PSIO_ZERO;
         psio_address next_Spi = PSIO_ZERO;
         for (int A = 0; A < 3*natom; A++) {
-            psio_->write(PSIF_HESS,"Smi^A",(char*)Smixp[0],nso * nocc * sizeof(double),next_Smi,&next_Smi);
+            psio_->write(PSIF_HESS,"Smi^A",(char*)Smixp[0],(unsigned long) nso * nocc * sizeof(double),next_Smi,&next_Smi);
         }
         for (int A = 0; A < 3*natom; A++) {
-            psio_->write(PSIF_HESS,"Sai^A",(char*)Saip[0],nvir * nocc * sizeof(double),next_Sai,&next_Sai);
+            psio_->write(PSIF_HESS,"Sai^A",(char*)Saip[0],(unsigned long) nvir * nocc * sizeof(double),next_Sai,&next_Sai);
         }
         for (int A = 0; A < 3*natom; A++) {
-            psio_->write(PSIF_HESS,"Sij^A",(char*)Sijp[0],nocc * nocc * sizeof(double),next_Sij,&next_Sij);
+            psio_->write(PSIF_HESS,"Sij^A",(char*)Sijp[0],(unsigned long) nocc * nocc * sizeof(double),next_Sij,&next_Sij);
         }
         for (int A = 0; A < 3*natom; A++) {
-            psio_->write(PSIF_HESS,"Spi^A",(char*)Spip[0],nmo * nocc * sizeof(double),next_Spi,&next_Spi);
+            psio_->write(PSIF_HESS,"Spi^A",(char*)Spip[0],(unsigned long) nmo * nocc * sizeof(double),next_Spi,&next_Spi);
```

Comment:
and the `unsigned long`s can be `size_t`, I think

Reply:
updated to `size_t`